OpenRA.Mods.Common/Traits/Render/Hovers.cs

```diff
@@ -21,21 +21,23 @@ namespace OpenRA.Mods.Common.Traits.Render
 	[Desc("Changes the visual Z position periodically.")]
 	public class HoversInfo : ConditionalTraitInfo, Requires<IMoveInfo>
 	{
-		[Desc("Amount of Z axis changes in world units.")]
-		public readonly int OffsetModifier = -43;
+		[Desc("Amount of Z axis changes in world units. Must be a negative value!")]
+		public readonly WDist OffsetModifier = new WDist(-43);
```

Comment:
Can we rename this to something like `BobHeight`? Our standard convention is to only use "Modifier" to mean a percentage multiplier.